src/BestBooks.js

```diff
@@ -25,11 +28,47 @@ class BestBooks extends React.Component {
       console.log(error)
     }
   }
+  
 
   componentDidMount() {
     this.fetchBooks();
   }
+  
+  
+  makeBook = async (bookMaker) => {
+    try {
+      const bookResponse = await axios.post(url + '/books:id', bookMaker);
+      this.setState({books: [...this.state.books, bookResponse.data]});
+      <Carousel.Item>{bookResponse.data.title}: {bookResponse.data.description}</Carousel.Item>
+    }catch (e) {
+      console.error(e)
+    }
+  }
+
+  deleteBook = async (id) => {
+    try {
+    await axios.delete(url + '/books/' + id);
+
+    const updatedBooks = this.state.books.filter(book => book._id !== id)
+    this.setState({ books: updatedBooks})
+  } catch (e) {
+    console.error(e)
+  }
```

Comment:
- in the `catch` block we can change `e` to `err` or `error` since `e` typically is passed to represent and event 👍